src/stories/card.stories.mdx

```diff
@@ -0,0 +1,35 @@
+import Card from '../components/Card/Card';
```

Comment:
Apologies for the confusion but there are 2 Cards component (for now).

This is the mentor card in the [mentors list](https://github.com/Coding-Coach/find-a-mentor/blob/4349034bc427f02871b1f5b87eb62c5da44592c2/src/components/Card/Card.js)

<img width="326" alt="Screen Shot 2021-01-06 at 23 17 49" src="https://user-images.githubusercontent.com/3723951/103820703-789ec980-5075-11eb-90de-ed2cddd61486.png">

while the one above is the card in the [back office](https://github.com/Coding-Coach/find-a-mentor/blob/3cfa9c582099fe6636aa1d80389efbe11bb24eeb/src/Me/components/Card/index.js).

<img width="394" alt="Screen Shot 2021-01-06 at 23 17 58" src="https://user-images.githubusercontent.com/3723951/103820697-763c6f80-5075-11eb-99c5-5803e6a65390.png">

Reply:
While I agree that there are two components called `Card` I am failing to understand what you're asking of me here.  Are you asking that I rename the story to something that makes it clear we're testing a Mentor Card as opposed to a Profile Card?

Reply:
Sorry.
I mean that you deleted the second card story (the one in the back office) but we need to keep it.

Reply:
Whoa, you are quite right.  My apologies. 😬

I put that story back under the name `profile-card.stories.js` to distinguish it from the mentor card.